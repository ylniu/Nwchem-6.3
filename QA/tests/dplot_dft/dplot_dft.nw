echo

start dplot_dft

geometry
  n  0 0   0.53879155
  n  0 0  -0.53879155
end

basis 
  n library sto-3g
end

task dft

dplot
  TITLE HOMO
   LimitXYZ
 -3.0 3.0 60
-3.0 3.0 60 
-3.0  3.0  60
  spin total density 
  gaussian
  output n2density.cube
end

task dplot

