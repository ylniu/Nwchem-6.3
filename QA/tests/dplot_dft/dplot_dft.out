 
Processor list
 
cu04n193,cu06n191
 
ARMCI configured for 2 cluster nodes. Network protocol is 'OpenIB Verbs API'.
All connections between all procs tested: SUCCESS
 argument  1 = /mscf/home/d3p852/nwchem-5.1.1/QA/tests/dplot_dft/dplot_dft.nw



============================== echo of input deck ==============================
echo

start dplot_dft

geometry
  n  0 0   0.53879155
  n  0 0  -0.53879155
end

basis
  n library sto-3g
end

task dft

dplot
  TITLE HOMO
   LimitXYZ
 -3.0 3.0 60
-3.0 3.0 60
-3.0  3.0  60
  spin total density
  gaussian
  output n2density.cube
end

task dplot

================================================================================


                                         
                                         


              Northwest Computational Chemistry Package (NWChem) 6.0
              ------------------------------------------------------


                    Environmental Molecular Sciences Laboratory
                       Pacific Northwest National Laboratory
                                Richland, WA 99352

                              Copyright (c) 1994-2010
                       Pacific Northwest National Laboratory
                            Battelle Memorial Institute

             NWChem is an open-source computational chemistry package
                        distributed under the terms of the
                      Educational Community License (ECL) 2.0
             A copy of the license is included with this distribution
                              in the LICENSE.TXT file

                                  ACKNOWLEDGMENT
                                  --------------

            This software and its documentation were developed at the
            EMSL at Pacific Northwest National Laboratory, a multiprogram
            national laboratory, operated for the U.S. Department of Energy
            by Battelle under Contract Number DE-AC05-76RL01830. Support
            for this work was provided by the Department of Energy Office
            of Biological and Environmental Research, Office of Basic
            Energy Sciences, and the Office of Advanced Scientific Computing.


           Job information
           ---------------

    hostname      = cu4n193
    program       = /scratch/nwchem
    date          = Fri Oct 29 11:36:18 2010

    compiled      = Thu_Oct_28_07:10:53_2010
    source        = /home/scicons/user/kurt/nwchem-6.0-release-pgf90-final/
    nwchem branch = 6.0
    input         = /mscf/home/d3p852/nwchem-5.1.1/QA/tests/dplot_dft/dplot_dft.nw
    prefix        = dplot_dft.
    data base     = ./dplot_dft.db
    status        = startup
    nproc         =        8
    time left     = 107964s



           Memory information
           ------------------

    heap     =  104857601 doubles =    800.0 Mbytes
    stack    =  104857601 doubles =    800.0 Mbytes
    global   =  209715200 doubles =   1600.0 Mbytes (distinct from heap & stack)
    total    =  419430402 doubles =   3200.0 Mbytes
    verify   = yes
    hardfail = no 


           Directory information
           ---------------------

  0 permanent = .
  0 scratch   = .




                                NWChem Input Module
                                -------------------



 Scaling coordinates for geometry "geometry" by  1.889725989
 (inverse scale =  0.529177249)

 ORDER OF PRIMARY AXIS IS BEING SET TO 4
 D4H symmetry detected

          ------
          auto-z
          ------


                             Geometry "geometry" -> ""
                             -------------------------

 Output coordinates in angstroms (scale by  1.889725989 to convert to a.u.)

  No.       Tag          Charge          X              Y              Z
 ---- ---------------- ---------- -------------- -------------- --------------
    1 n                    7.0000     0.00000000     0.00000000     0.53879155
    2 n                    7.0000     0.00000000     0.00000000    -0.53879155

      Atomic Mass 
      ----------- 

      n                 14.003070


 Effective nuclear repulsion energy (a.u.)      24.0628172444

            Nuclear Dipole moment (a.u.) 
            ----------------------------
        X                 Y               Z
 ---------------- ---------------- ----------------
     0.0000000000     0.0000000000     0.0000000000

      Symmetry information
      --------------------

 Group name             D4h       
 Group number             28
 Group order              16
 No. of unique centers     1

      Symmetry unique atoms

     1



                                Z-matrix (autoz)
                                -------- 

 Units are Angstrom for bonds and degrees for angles

      Type          Name      I     J     K     L     M      Value
      ----------- --------  ----- ----- ----- ----- ----- ----------
    1 Stretch                  1     2                       1.07758


            XYZ format geometry
            -------------------
     2
 geometry
 n                     0.00000000     0.00000000     0.53879155
 n                     0.00000000     0.00000000    -0.53879155

 ==============================================================================
                                internuclear distances
 ------------------------------------------------------------------------------
       center one      |      center two      | atomic units |  angstroms
 ------------------------------------------------------------------------------
    2 n                |   1 n                |     2.03634  |     1.07758
 ------------------------------------------------------------------------------
                         number of included internuclear distances:          1
 ==============================================================================



  library name resolved from: environment
  library file name is: <
 /mscf/scicons/apps/nwchem-6.0.oct19//src/basis/libraries/>
  
                      Basis "ao basis" -> "" (cartesian)
                      -----
  n (Nitrogen)
  ------------
            Exponent  Coefficients 
       -------------- ---------------------------------------------------------
  1 S  9.91061690E+01  0.154329
  1 S  1.80523120E+01  0.535328
  1 S  4.88566020E+00  0.444635

  2 S  3.78045590E+00 -0.099967
  2 S  8.78496600E-01  0.399513
  2 S  2.85714400E-01  0.700115

  3 P  3.78045590E+00  0.155916
  3 P  8.78496600E-01  0.607684
  3 P  2.85714400E-01  0.391957



 Summary of "ao basis" -> "" (cartesian)
 ------------------------------------------------------------------------------
       Tag                 Description            Shells   Functions and Types
 ---------------- ------------------------------  ------  ---------------------
 n                           sto-3g                  3        5   2s1p



                                 NWChem DFT Module
                                 -----------------


  Caching 1-el integrals 

            General Information
            -------------------
          SCF calculation type: DFT
          Wavefunction type:  closed shell.
          No. of atoms     :     2
          No. of electrons :    14
           Alpha electrons :     7
            Beta electrons :     7
          Charge           :     0
          Spin multiplicity:     1
          Use of symmetry is: on ; symmetry adaption is: on 
          Maximum number of iterations:  30
          AO basis - number of functions:    10
                     number of shells:     6
          Convergence on energy requested: 1.00D-06
          Convergence on density requested: 1.00D-05
          Convergence on gradient requested: 5.00D-04

              XC Information
              --------------
                        Slater Exchange Functional  1.000 local    
                      VWN V Correlation Functional  1.000 local    

             Grid Information
             ----------------
          Grid used for XC integration:  medium    
          Radial quadrature: Mura-Knowles        
          Angular quadrature: Lebedev. 
          Tag              B.-S. Rad. Rad. Pts. Rad. Cut. Ang. Pts.
          ---              ---------- --------- --------- ---------
          n                   0.65       49           5.0       434
          Grid pruning is: on 
          Number of quadrature shells:    49
          Spatial weights used:  Erf1

          Convergence Information
          -----------------------
          Convergence aids based upon iterative change in 
          total energy or number of iterations. 
          Levelshifting, if invoked, occurs when the 
          HOMO/LUMO gap drops below (HL_TOL): 1.00D-02
          DIIS, if invoked, will attempt to extrapolate 
          using up to (NFOCK): 10 stored Fock matrices.

                    Damping( 0%)  Levelshifting(0.5)       DIIS
                  --------------- ------------------- ---------------
          dE  on:    start            ASAP                start   
          dE off:    2 iters         30 iters            30 iters 


      Screening Tolerance Information
      -------------------------------
          Density screening/tol_rho: 1.00D-10
          AO Gaussian exp screening on grid/accAOfunc:  14
          CD Gaussian exp screening on grid/accCDfunc:  20
          XC Gaussian exp screening on grid/accXCfunc:  20
          Schwarz screening/accCoul: 1.00D-08


      Superposition of Atomic Density Guess
      -------------------------------------

 Sum of atomic energies:        -107.23242776

      Non-variational initial energy
      ------------------------------

 Total energy =    -107.692522
 1-e energy   =    -194.898780
 2-e energy   =      63.143441
 HOMO         =      -0.292719
 LUMO         =       0.219084


      Symmetry analysis of molecular orbitals - initial
      -------------------------------------------------

  Numbering of irreducible representations: 

     1 a1g         2 a1u         3 a2g         4 a2u         5 b1g     
     6 b1u         7 b2g         8 b2u         9 eg         10 eu      

  Orbital symmetries:

     1 a1g         2 a2u         3 a1g         4 a2u         5 eu      
     6 eu          7 a1g         8 eg          9 eg         10 a2u     

   Time after variat. SCF:      0.6
   Time prior to 1st pass:      0.6

 #quartets = 1.550D+02 #integrals = 4.010D+02 #direct =  0.0% #cached =100.0%


 Integral file          = ./dplot_dft.aoints.0
 Record size in doubles =  65536        No. of integs per rec  =  43688
 Max. records in memory =      2        Max. records in file   = 169376
 No. of bits per label  =      8        No. of bits per value  =     64


File balance: exchanges=     0  moved=     0  time=   0.0


 Grid_pts file          = ./dplot_dft.gridpts.0
 Record size in doubles =  12289        No. of grid_pts per rec  =   3070
 Max. records in memory =      2        Max. recs in file   =    903273


           Memory utilization after 1st SCF pass: 
           Heap Space remaining (MW):      104.70           104701073
          Stack Space remaining (MW):      104.86           104857439

   convergence    iter        energy       DeltaE   RMS-Dens  Diis-err    time
 ---------------- ----- ----------------- --------- --------- ---------  ------
 d= 0,ls=0.0,diis     1   -107.1321690083 -1.31D+02  2.30D-02  8.98D-02     0.7
 d= 0,ls=0.0,diis     2   -107.1349102115 -2.74D-03  3.01D-03  1.55D-04     0.7
 d= 0,ls=0.0,diis     3   -107.1349344218 -2.42D-05  2.54D-04  1.18D-06     0.7
 d= 0,ls=0.0,diis     4   -107.1349346270 -2.05D-07  6.10D-08  7.25D-14     0.7


         Total DFT energy =     -107.134934627023
      One electron energy =     -193.242984118910
           Coulomb energy =       74.843756992113
    Exchange-Corr. energy =      -12.798524744655
 Nuclear repulsion energy =       24.062817244429

 Numeric. integr. density =       14.000000077438

     Total iterative time =      0.1s



                       DFT Final Molecular Orbital Analysis
                       ------------------------------------

 Vector    1  Occ=2.000000D+00  E=-1.382408D+01  Symmetry=a2u
              MO Center=  0.0D+00,  0.0D+00,  7.0D-19, r^2= 3.1D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     1     -0.701066  1 N  s                  6      0.701066  2 N  s          

 Vector    2  Occ=2.000000D+00  E=-1.382380D+01  Symmetry=a1g
              MO Center=  2.4D-17, -3.0D-18,  2.6D-17, r^2= 3.1D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     1      0.701768  1 N  s                  6      0.701768  2 N  s          

 Vector    3  Occ=2.000000D+00  E=-1.058327D+00  Symmetry=a1g
              MO Center= -6.3D-18, -2.6D-17, -3.5D-18, r^2= 4.8D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     2      0.461063  1 N  s                  7      0.461063  2 N  s          
     5     -0.275973  1 N  pz                10      0.275973  2 N  pz         
     6     -0.177458  2 N  s                  1     -0.177458  1 N  s          

 Vector    4  Occ=2.000000D+00  E=-4.183250D-01  Symmetry=a2u
              MO Center=  9.5D-34,  2.4D-34,  1.9D-16, r^2= 1.2D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     2      0.771664  1 N  s                  7     -0.771664  2 N  s          
     5      0.238387  1 N  pz                10      0.238387  2 N  pz         
     1     -0.178185  1 N  s                  6      0.178185  2 N  s          

 Vector    5  Occ=2.000000D+00  E=-3.996932D-01  Symmetry=eu
              MO Center=  5.7D-17,  2.9D-16,  5.6D-16, r^2= 8.3D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     4      0.610158  1 N  py                 9      0.610158  2 N  py         

 Vector    6  Occ=2.000000D+00  E=-3.996932D-01  Symmetry=eu
              MO Center= -2.2D-17,  4.5D-18, -3.5D-18, r^2= 8.3D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     3      0.610158  1 N  px                 8      0.610158  2 N  px         

 Vector    7  Occ=2.000000D+00  E=-2.837795D-01  Symmetry=a1g
              MO Center= -5.2D-17, -2.6D-16,  1.1D-16, r^2= 1.2D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     5     -0.590545  1 N  pz                10      0.590545  2 N  pz         
     7     -0.436474  2 N  s                  2     -0.436474  1 N  s          

 Vector    8  Occ=0.000000D+00  E= 1.233395D-02  Symmetry=eg
              MO Center= -3.6D-19, -1.1D-21, -5.7D-17, r^2= 8.8D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     8     -0.840090  2 N  px                 3      0.840090  1 N  px         

 Vector    9  Occ=0.000000D+00  E= 1.233395D-02  Symmetry=eg
              MO Center= -8.9D-21,  2.9D-18, -5.6D-16, r^2= 8.8D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     9      0.840090  2 N  py                 4     -0.840090  1 N  py         

 Vector   10  Occ=0.000000D+00  E= 7.530543D-01  Symmetry=a2u
              MO Center=  3.7D-19, -2.9D-18,  2.8D-16, r^2= 5.9D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     2     -1.250063  1 N  s                  7      1.250063  2 N  s          
     5      1.249408  1 N  pz                10      1.249408  2 N  pz         

  Format used          : Gaussian9x Cube

 center of mass
 --------------
 x =   0.00000000 y =   0.00000000 z =   0.00000000

 moments of inertia (a.u.)
 ------------------
          29.033037760134           0.000000000000           0.000000000000
           0.000000000000          29.033037760134           0.000000000000
           0.000000000000           0.000000000000           0.000000000000

     Multipole analysis of the density
     ---------------------------------

     L   x y z        total         alpha         beta         nuclear
     -   - - -        -----         -----         ----         -------
     0   0 0 0      0.000000     -7.000000     -7.000000     14.000000

     1   1 0 0      0.000000      0.000000      0.000000      0.000000
     1   0 1 0      0.000000      0.000000      0.000000      0.000000
     1   0 0 1      0.000000      0.000000      0.000000      0.000000

     2   2 0 0     -6.843287     -3.421644     -3.421644      0.000000
     2   1 1 0      0.000000      0.000000      0.000000      0.000000
     2   1 0 1      0.000000      0.000000      0.000000      0.000000
     2   0 2 0     -6.843287     -3.421644     -3.421644      0.000000
     2   0 1 1      0.000000      0.000000      0.000000      0.000000
     2   0 0 2     -8.628319    -11.570828    -11.570828     14.513336


 Parallel integral file used       8 records with       0 large values


 Task  times  cpu:        0.2s     wall:        0.3s


                                NWChem Input Module
                                -------------------


  trying scf 
  trying dft 
  File vec is 
 ./dplot_dft.movecs                                                              

  Limits (a.u.) specified for the density plot:
  ---------------------------------------------

        From        To      # of spacings
X    -5.66918     5.66918        60
Y    -5.66918     5.66918        60
Z    -5.66918     5.66918        60

  Total number of grid points =                    226981

  1-st set of MOs      : ./dplot_dft.movecs
  Output is written to : n2density.cube
  Format used          : Gaussian9x Cube
  Type of picture      : CHARGE DENSITY
  Format used          : Gaussian9x Cube
  Spin                 : TOTAL   
  The density is computed using all orbitals
  The density is computed on the specified grid
  Format used          : Gaussian9x Cube
  Format used          : Gaussian9x Cube
  Format used          : Gaussian9x Cube
  Format used          : Gaussian9x Cube
  Format used          : Gaussian9x Cube
  max element     76.57872976653427     

  Aproximate Charge    =     0.00

 Task  times  cpu:        0.5s     wall:        0.5s


                                NWChem Input Module
                                -------------------


 Summary of allocated global arrays
-----------------------------------
  No active global arrays



                         GA Statistics for process    0
                         ------------------------------

       create   destroy   get      put      acc     scatter   gather  read&inc
calls:  138      138     2681      289      180        0        0        0     
number of processes/call 1.17e+00 1.36e+00 3.26e+00 0.00e+00 0.00e+00
bytes total:             6.42e+05 8.32e+04 9.84e+04 0.00e+00 0.00e+00 0.00e+00
bytes remote:            5.59e+04 1.51e+04 3.87e+04 0.00e+00 0.00e+00 0.00e+00
Max memory consumed for GA by this process: 228584 bytes

MA_summarize_allocated_blocks: starting scan ...
MA_summarize_allocated_blocks: scan completed: 0 heap blocks, 0 stack blocks
MA usage statistics:

	allocation statistics:
					      heap	     stack
					      ----	     -----
	current number of blocks	         0	         0
	maximum number of blocks	        24	        50
	current total bytes		         0	         0
	maximum total bytes		   1252224	  22509848
	maximum total K-bytes		      1253	     22510
	maximum total M-bytes		         2	        23


                                     CITATION
                                     --------
                Please cite the following reference when publishing
                           results obtained with NWChem:

                 M. Valiev, E.J. Bylaska, N. Govind, K. Kowalski,
              T.P. Straatsma, H.J.J. van Dam, D. Wang, J. Nieplocha,
                        E. Apra, T.L. Windus, W.A. de Jong
                 "NWChem: a comprehensive and scalable open-source
                  solution for large scale molecular simulations"
                      Comput. Phys. Commun. 181, 1477 (2010)
                           doi:10.1016/j.cpc.2010.04.018

                              AUTHORS & CONTRIBUTORS
                              ----------------------
      E. J. Bylaska, W. A. de Jong, N. Govind, K. Kowalski, T. P. Straatsma,
     M. Valiev, H. J. J. van Dam, D. Wang, E. Apra, T. L. Windus, J. Hammond,
     J. Autschbach, P. Nichols, S. Hirata, M. T. Hackler, Y. Zhao, P.-D. Fan,
      R. J. Harrison, M. Dupuis, D. M. A. Smith, K. Glaesemann, J. Nieplocha,
      V. Tipparaju, M. Krishnan, A. Vazquez-Mayagoitia, L. Jensen, M. Swart,
      Q. Wu, T. Van Voorhis, A. A. Auer, M. Nooijen, L. D. Crosby, E. Brown,
             G. Cisneros, G. I. Fann, H. Fruchtl, J. Garza, K. Hirao,
        R. Kendall, J. A. Nichols, K. Tsemekhman, K. Wolinski, J. Anchell,
       D. Bernholdt, P. Borowski, T. Clark, D. Clerc, H. Dachsel, M. Deegan,
        K. Dyall, D. Elwood, E. Glendening, M. Gutowski, A. Hess, J. Jaffe,
        B. Johnson, J. Ju, R. Kobayashi, R. Kutteh, Z. Lin, R. Littlefield,
    X. Long, B. Meng, T. Nakajima, S. Niu, L. Pollack, M. Rosing, G. Sandrone,
       M. Stave, H. Taylor, G. Thomas, J. H. van Lenthe, A. Wong, Z. Zhang.

 Total times  cpu:        0.7s     wall:        2.4s
